git-tl

```diff
@@ -375,9 +381,9 @@ def sync(include_all, no_fetch):
     if include_all:
         roots = [b for b in branches if b.parent is None]
     else:
-        roots = [b for b in branches if b.name == 'origin/main']
+        roots = [b for b in branches if b.name == default_upstream]
```

Comment:
If the default_upstream is 'main', we want to actually get 'origin/main' here to preserve the current behavior. I'm not sure that's actually necessary, though. (Yikes, do I need to start adding tests to this repo!?)